### DISCUSSION TITLE
AnvilLoader path

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#1209] [Help]
Hi, is there a way to get the path the anvilloader is using? I want to check which world the anvilloader is using for the instance of the player.
~Jochem

**Comment by RinesThaix** [comment]
Hi. You pass the path to the AnvilLoader on it's instantiation. That's where the world directory is located.

**Reply by DevBlokJochem** [reply]
But how can I get the path? So for example I want to check if the player is in the lobby, but I have multiple lobbys. So I have to check if the path of the instance from the player is the same as the path where the lobby is located.

**Reply by RinesThaix** [reply]
1. You create instance independently of AnvilLoader. What loader does is just populates given instance container.
2. When you instantiate an instance, you can specify UUID. Nothing prevents you from doing something like `uuid = UUID.nameUUIDFromBytes(myWorldName.toByteArray())`. It already allows you to keep a `Map<UUID, Instance>` so that you could retrieve an instance by the world name. If it's not enough and you want to get a world name by the instance itself, you can create your own instance container implementation and save it as a field.

**Reply by DevBlokJochem** [reply]
👍
